source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -243,6 +243,13 @@ Use this method to safely install |CL| on media with available space, or
 alongside existing partitions, and accept the `Default partition schema`_.
 If enough free space exists, safe installation is allowed.
 
+.. note::
+
+   |CL| allows installation along-side another OS. During BIOS POST stage,
+   you can select a bootable device/partitions on a drive. Some BIOSes can
+   present multiple bootable partitions during this stage, but some will
+   show only one bootable partition on a drive. Be aware of limitations.
```

Comment:
This is how I would word it:

|CL| allows installation along-side another OS.  Typically, when you boot your system, you can list and select a bootable device or partition during the BIOS POST stage by pressing an F key.  Some BIOSes may present the |CL| partition and you can select and boot it.  However, there are some BIOSes that may only show the primary partition and you will not be able boot |CL| in this case.  Be aware of this possible limitation with some BIOSes.